### PR TITLE
Fix minor UG bugs

### LINKED ISSUE #221: UG Bugs
- Point 1 is repeated twice.
image

- Image description doesn't follow format.
image

- Formatting failed.
image

- tau_bar is invalid friend Id
image

- Heading followed by a :
image
image

### PR DESCRIPTION
**Pull request by tau-bar**
# Changes:
* Fix minor UG bugs

#Issues:
* Resolves #221

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -435,12 +434,12 @@ e.g. `Apex Legends` may be stored as `ApexLegends`.
 </div>
 
 Examples:
-* `game --add Valorant` Adds a brand-new game into the game list with the **unique** `GAME_ID` 'Valorant'.
+* `game --add RocketLeague` Adds a brand-new game into the game list with the **unique** `GAME_ID` 'RocketLeague'.
```
Comment (YeluriKetan) [inline docs/UserGuide.md:437]: All other references and mentions of the game RocketLeague in the captions are in `code`. Can perhaps convert this too.
'RocketLeague' -> `RocketLeague`

**Review by YeluriKetan — CHANGES_REQUESTED** [review]
Tiny fix needed. Other than that LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (tau-bar) [reply docs/UserGuide.md:437]: I think for the other commands, the name of the game or the friend's ID is also in '....'. If want to update to `code` have to update the rest as well.

Example in `game --delete`:
Screenshot 2021-11-07 at 1 42 33 PM

Comment (YeluriKetan) [reply docs/UserGuide.md:437]: Interesting. It appears that in the examples the gameId and friendId are in 'single quotes' but in the image captions, the gameId and friendId are in `code` .

**Review by YeluriKetan — APPROVED** [review]
LGTM.

**Pull request merged**